dali/test/python/test_torch_pipeline_rnnt.py

```diff
@@ -269,25 +282,31 @@ def test_rnnt_data_pipeline():
     comparing to the reference: random operations (i.e. dither and presampling
     aka "speed perturbation") are turned off
     """
-    batch_size = 2
     ref_pipeline = FilterbankFeatures(sample_rate=16000, n_fft=512, highfreq=.0, dither=.00001,
                                       frame_splicing=3)
-    data_path = os.path.join(dali_extra_path, "db", "audio", "rnnt_data_pipeline")
-    rec_names = ["and_showed_itself_decoded.npy", "asked_her_father_decoded.npy"]
-    recordings = [np.load(os.path.join(data_path, name)) for name in rec_names]
-    pipe = RnntTrainPipeline(device_id=0, n_devices=1, file_root=data_path,
-                             file_list=os.path.join(data_path, "file_list.txt"),
-                             batch_size=batch_size)
+    recordings = []
+    for fpath in npy_files:
+        arr = np.load(fpath)
+        arr = _convert_samples_to_float32(arr)
+        recordings.append(arr)
+    batch_size = len(recordings)
```

Comment:
How many samples it could be?
Maybe we should have a fixed batch size and just use it instead of connecting it with the number of samples?

Reply:
it's 43 samples now.

Reply:
Should be bearable.